### PR TITLE
Modularize Composer scripts

### PR DESCRIPTION
**Pull request by szepeviktor**
Just that :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -55,7 +55,12 @@
         }
     },
     "scripts": {
-        "test": "./vendor/bin/pest --parallel && ./vendor/bin/phpstan analyse"
+        "test:pest": "pest --parallel",
```
Comment (danharrin) [inline composer.json:58]: Don't we still need the `./vendor/bin` prefix though?

Comment (szepeviktor) [reply composer.json:58]: Composer is witty: puts `./vendor/bin` in the beginning of `$PATH`
https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands (purple callout)
Even on Windows!

Comment (danharrin) [reply composer.json:58]: Awesome, TIL!

**Comment by szepeviktor** [comment]
Thank you.

**Pull request merged**